packages/react-components/react-table/src/hooks/useTableColumnResizeMouseHandler.ts

```diff
@@ -2,6 +2,27 @@ import * as React from 'react';
 import { TableColumnId, ColumnResizeState } from './types';
 import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 
+type TouchOrMouseEvent = MouseEvent | TouchEvent;
+type ReactTouchOrMouseEvent = React.MouseEvent | React.TouchEvent;
+
+function isTouchEvent(event: TouchOrMouseEvent | ReactTouchOrMouseEvent): event is TouchEvent {
+  return ['touchstart', 'touchmove', 'touchend'].includes(event.type);
+}
+
+function isMouseEvent(event: TouchOrMouseEvent | ReactTouchOrMouseEvent): event is MouseEvent {
+  return ['mousedown', 'mousemove', 'mouseup'].includes(event.type);
+}
+
+function getEventClientX(event: TouchOrMouseEvent | ReactTouchOrMouseEvent): number {
+  if (isMouseEvent(event)) {
+    return event.clientX;
+  } else if (isTouchEvent(event)) {
+    return event.touches[0].clientX;
+  } else {
+    throw new Error('Unable to get clientX. Unknown event type.');
```

Comment:
is it safe to throw for apps in production? I'd wrap this with a development check